react-native-cli/index.js

```diff
@@ -198,7 +199,14 @@ function run(root, projectName, logLevel) {
   if (logLevel === 'debug' || logLevel === 'verbose') {
     spawnArgs = {stdio: 'inherit'};
   }
-  var proc = spawn('npm', args, spawnArgs);
+  var proc;
+  if (os.platform() === 'win32'){
+    args.unshift('npm');
+    args.unshift('/c');
+    proc = spawn('cmd', args, spawnArgs);
```

Comment:
`cmd /c` is just same as `bash -c` on *nix
spawn cannot run a .cmd file(like .sh on *nix) on windows, but we can invoke it through cmd.exe